### PR TITLE
Special button

### PR DESCRIPTION
**Pull request by honda0306**
# Description

Created SpecialButton component that is used for secondary functions (add new cards, etc). Altered Button and LandingPage styling to reflect design file.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Change status

- [x] Complete, tested, ready to review and merge

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: frontend/src/pages/Landingpage.tsx [hunk 1]
```diff
@@ -46,6 +57,12 @@ const SideHeader = styled('h3')`
...
   display: flex;
+  justify-content: space-evenly;
+  position: fixed;
+  left: 0px;
+  bottom: 0px;
+  height: 10%;
+  width: 100%;
 `
 
 type Props = {};
```
Comment (NandoTheessen) [inline frontend/src/pages/Landingpage.tsx]: This should be removed if you are not using it.

File: frontend/src/components/shared_components/index.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import Button from './Button';
+import SpecialButton from './SpecialButton';
 // import Form from './Form';
 
 export { Button };
```
Comment (NandoTheessen) [inline frontend/src/components/shared_components/index.ts]: should be export { Button, SpecialButton };

File: frontend/src/components/shared_components/SpecialButton.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { string } from 'prop-types';
```
Comment (NandoTheessen) [inline frontend/src/components/shared_components/SpecialButton.tsx]: We are not using prop-types, they shouldn't even be in the dependency tree. 
Did you add them? 
If yes, remove them please.

Comment (honda0306) [reply frontend/src/components/shared_components/SpecialButton.tsx]: I didn't add them, but I'm happy to remove 😁

**Pull request merged**